src/app/app-routing.module.ts

```diff
@@ -385,6 +385,12 @@ export class AuthGuard implements CanActivate {
         }
       ]
     },
+    {
+      path: 'todo-list',
+      pathMatch: 'full',
+      canActivate: [AuthGuard],
+      loadChildren: () => import('./todo-list/todo-list.module').then(m => m.TodoListModule)
```

Comment:
lazy loaded module, so anything I add in this route will not bloat the load times for the rest of the app. Could be useful when I bring in ag-grid